c2tc-mobile/components/PanelComponent/ButtonInterface.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Button, StyleSheet, View, Dimensions } from "react-native";
+
+const { width, height } = Dimensions.get("window");
+
+export default class ButtonInterace extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      type: this.props.type
+    };
+  }
+
+  componentDidMount() {}
+
+  getType() {
+    return this.state.type;
+  }
+
+  _onPressLearnMore() {
```

Comment:
Make this the same function that is in LiveLocation so that only a single function is called. Right now both onPress() are called